### ISSUE TITLE
I'm trying to integrate HTML Render into my Open Source project Tangenta40

### ISSUE THREAD

**Issue by dstrucl** [ArthurHub/HTML-Renderer#78]
I'm developing open source project https://github.com/dstrucl/Tangenta40.
I'm trying to use your HTML-Renderer for printnig Invoices and Proforma-Invoices.
There is a problem to print a big Invoice with many items on multiple A4 pages.
I'm going to modify some source code in your HTML renderer.
Can you give me an advice.
I' have allready integrated Source code of your HTML-Renderer into Tangenta40 project.

**Comment by bjorn-ali-goransson** [comment]
Provide some HTML example?

**Comment by dstrucl** [comment]
You can find samples in https://github.com/dstrucl/Tangenta40/tree/master/TANGENTA/Tangenta_DefaultPrintTemplates
These are samples with tokens (keys) that are replaced by real values and then printed.
I have changes Arthurs's rendererand you can find it at:
https://github.com/dstrucl/Tangenta40/tree/master/HtmlRenderer%201.5.0.6%20with%20print%20pages%20support
 Arthurs's Html renderer needs many improvements and it is not working so well. It is also not comaptible with major source browse renders.
Regards Damjan

**Comment by bjorn-ali-goransson** [comment]
Yes, this software is very old and hacky. There is another fork of this project that seems to be more up to date:

https://github.com/ggeurts/HTML-Renderer

Also other forks exist, maybe you can explore them. The creator of AngleSharp was planning to make a similar library as this one, hope it can be released soon.

As to your examples I didn't really understand what would be the problem with them. You need to be more specific if you want any help.
